src/pre_defined_classes/mathematical_program/hard/q3/MainTest.java

```diff
@@ -0,0 +1,27 @@
+package pre_defined_classes.mathematical_program.hard.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    //Parsons
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The area of the sector is: "),
+                new DoubleLiteral("math")
+        };
+    }
+
+    public void runMain() {
+        AreaSector.main(new String[]{});
+    }
+
+    @Test
+    public void mathTest() {
+        assertEquals(24.801128670839425, Double.parseDouble(getItemByName("math")), 0.1, "Your program's math calculations are incorrect");
```

Comment:
This error message could be improved a bit. Maybe more like:
"Your calculated sector area is incorrect."

Reply:
Why? They're only doing the one calculation, the output already says that its the calculated sector area